mkdocs/docs/en/changelog/0.10.0.md

```diff
@@ -0,0 +1,535 @@
+# 0.10.0 (2023-12-18)
+
+## Breaking Changes
+
+- Upgrade `etl-entities` from v1 to v2 ([#172](https://github.com/MobileTeleSystems/onetl/pull/172)).
+
+  This implies that `HWM` classes are now have different internal structure than they used to.
+
+  Before:
+
+  ```python
+  from etl_entities.old_hwm import IntHWM as OldIntHWM
+  from etl_entities.source import Column, Table
+  from etl_entities.process import Process
+
+  hwm = OldIntHWM(
+      process=Process(name="myprocess", task="abc", dag="cde", host="myhost"),
+      source=Table(name="schema.table", instance="postgres://host:5432/db"),
+      column=Column(name="col1"),
+      value=123,
+  )
+  ```
+
+  After:
+
+  ```python
+  from etl_entities.hwm import ColumnIntHWM
+
+  hwm = ColumnIntHWM(
+      name="some_unique_name",
+      description="any value you want",
+      source="schema.table",
+      expression="col1",
+      value=123,
+  )
+  ```
+
+  **Breaking change:** If you used HWM classes from `etl_entities` module, you should rewrite your code to make it compatible with new version.
+
+??? "More details"
+
+    - ``HWM`` classes used by previous onETL versions were moved from ``etl_entities`` to ``etl_entities.old_hwm`` submodule. They are here for compatibility reasons, but are planned to be removed in ``etl-entities`` v3 release.
+    - New ``HWM`` classes have flat structure instead of nested.
+    - New ``HWM`` classes have mandatory ``name`` attribute (it was known as ``qualified_name`` before).
+    - Type aliases used while serializing and deserializing ``HWM`` objects to ``dict`` representation were changed too: ``int`` → ``column_int``.
+
+
+  To make migration simpler, you can use new method:
+
+  ```python
+  old_hwm = OldIntHWM(...)
+  new_hwm = old_hwm.as_new_hwm()
+  ```
+
+  Which automatically converts all fields from old structure to new one, including `qualified_name` → `name`.
+
+- **Breaking changes:**
+
+  - Methods `BaseHWMStore.get()` and `BaseHWMStore.save()` were renamed to `get_hwm()` and `set_hwm()`.
+  - They now can be used only with new HWM classes from `etl_entities.hwm`, **old HWM classes are not supported**.
+
+  If you used them in your code, please update it accordingly.
+
+- YAMLHWMStore **CANNOT read files created by older onETL versions** (0.9.x or older).
+
+??? "Update procedure"
+
+    ```python
```

Comment:
Replace 4 spaces indentation with 2 spaces to enable syntax highlighting, same for blocks below